service/create/service.go

```diff
@@ -865,6 +866,19 @@ func (s *Service) onAdd(obj interface{}) {
 	}
 	ingressHZID := ingressHZ.GetID()
 
+	wildcardHZInput := hostedZoneInput{
+		Cluster: cluster,
+		Domain:  cluster.Spec.Cluster.Kubernetes.IngressController.WildcardDomain,
+		Client:  clients.Route53,
+	}
+
+	wildcardHZ, err := s.createHostedZone(wildcardHZInput)
```

Comment:
This should use the Ingress HZ in awstpr rather than creating it. See the changes in https://github.com/giantswarm/aws-operator/pull/284